function/filter/filter.go

```diff
@@ -35,6 +35,9 @@ func (list filterList) Less(i, j int) bool {
 	if math.IsNaN(list.value[j]) && !math.IsNaN(list.value[i]) {
 		return true
 	}
+	if math.IsNaN(list.value[j]) {
```

Comment:
Not exactly sure how we're defining, but isn't the `if` block above now useless?